src/main/java/org/gridsuite/explore/server/services/ExploreService.java

```diff
@@ -204,6 +204,17 @@ public void deleteElement(UUID id, String userId) {
         }
     }
 
+    public void deleteElements(List<UUID> uuids, String userId) {
+        try {
+            uuids.forEach(id -> directoryService.deleteElement(id, userId));
+            directoryService.deleteDirectoryElements(uuids, userId);
+            // FIXME dirty fix to ignore errors and still delete the elements in the directory-server. To delete when handled properly.
```

Comment:
What prevents us to handle this properly at this time ?

Reply:
I used the same logic of this PR: https://github.com/gridsuite/explore-server/pull/27. we ignore the exception and do the deletion anyway